packages/virgo/src/services/event.ts

```diff
@@ -158,7 +158,10 @@ export class VirgoEventService<TextAttributes extends BaseTextAttributes> {
           node => node instanceof HTMLElement
         ).length === 1
       ) {
-        this._editor.focusEnd();
+        this._editor.rangeService.setVRange({
+          index: this._editor.yText.length,
+          length: range.endOffset,
+        });
```

Comment:
@Flrande  When backward selection, the cursor will jump, start <-> end, very unstable, so changed to the current

Reply:
What is the value of native range in this case? It seems that this change will make vRange overflow because the index has been reached max value (this._editor.yText.length).

Reply:
<img width="561" alt="Screenshot 2023-05-10 at 9 12 09 AM" src="https://github.com/toeverything/blocksuite/assets/27926/d7a3dd85-5d4f-4fda-b353-356d0f160b4d">